### DISCUSSION TITLE
Real-time sync on input

### DISCUSSION THREAD

**Discussion by magheralex** [mdbassit/Coloris#167] [General]
Hi there,
First of all, it is a great picker. Congratulations!

However, I use format: 'hex' and would like to sync the input with the picker in real time. Is it possible? Now i have to click the color bullet or outside in order to apply the input hex.

For example, if i type #800080, I want to apply the color (to clr-color-area, clr-hue and clr-alpha) without any additional click.

**Comment by mdbassit** [comment]
Hello and thank you for the kind words.

For performance reasons, you have to press the Enter key or click outside for the color that you type to be applied. This is unlikely to change. But if you want to force your desired behavior in your own version, try adding this code after `Coloris` is initialized:

```js
Coloris.ready(() => {
  const colorValue = document.getElementById('clr-color-value');
  
  colorValue.addEventListener('input', () => {
    colorValue.dispatchEvent(new Event('change'));
  });
});
```
